app/models/teleconsultation.rb

```diff
@@ -0,0 +1,42 @@
+class Teleconsultation < ApplicationRecord
+  include Mergeable
+
+  belongs_to :patient
+  belongs_to :medical_officer, class_name: "User", foreign_key: :medical_officer_id
+  belongs_to :facility, optional: true
+  belongs_to :requester, class_name: "User", foreign_key: :requester_id, optional: true
+
+  REQUEST_ATTRIBUTES = %w[requester_id facility_id requested_at request_completed]
```

Comment:
`freeze` the `REQUEST_ATTRIBUTES` and `RECORD_ATTRIBUTES` as well? 

Or add `# frozen_string_literal: true` on top of the file.